server/templates/base.html

```diff
@@ -29,7 +30,7 @@
 -#}
 <!DOCTYPE html>
 
-<html lang={{ locale }}>
+<html lang={{ locale }} class={{ DEPLOYMENT }}>
```

Comment:
i'd prefer it if we only set an overriding class in the private case, so public remains the default (besides some performance concerns, also to avoid us relying on the public class)

Reply:
Done